backend/zfs_ops.py

```
@@ -245,3 +314,149 @@ async def is_zfs_available(self) -> bool:
+            
+            # First argument is the command, rest are arguments
+            command = args[0]
+            command_args = args[1:] if len(args) > 1 else []
+            
+            cmd = SecurityUtils.validate_system_command_args(command, *command_args)
+            return await self.run_command(cmd)
+        except SecurityValidationError:
+            return 1, "", "Security validation failed"
+    
+    async def force_unmount_dataset(self, target_host: str, dataset_path: str, 
+                                   ssh_user: str = "root", ssh_port: int = 22) -> bool:
```

Flake8 (7.2.0)

[error] continuation line under-indented for visual indent

(E128)